test/Microsoft.Health.Fhir.Shared.Tests.Integration/Persistence/SqlServerFhirStorageTestsFixture.cs

```diff
@@ -84,7 +81,10 @@ public SqlServerFhirStorageTestsFixture()
             SqlTransactionHandler = new SqlTransactionHandler();
             SqlConnectionWrapperFactory = new SqlConnectionWrapperFactory(config, SqlTransactionHandler, new SqlCommandWrapperFactory());
 
-            SqlServerStatusRegistryDataStore = new SqlServerStatusRegistryDataStore(SqlConnectionWrapperFactory, upsertSearchParamsTvpGenerator);
+            SqlServerStatusRegistryDataStore = new SqlServerStatusRegistryDataStore(
+                () => SqlConnectionWrapperFactory.CreateMockScope(),
```

Comment:
@brendankowitz referencing a unit test project from the integration tests doesn't feel right.. Is this the best way to create a `Func<IScoped<SqlConnectionWrapperFactory>>()` object?

Reply:
Possibly we could move this extension to Microsoft.Health.Fhir.Tests.Common

Reply:
Moved in 4ab9daf and removed the old reference in ed5dc53.